frappe_io/www/docs/user/en/api/dialog.md

```diff
@@ -111,18 +156,169 @@ frappe.prompt([
 	console.log(values.first_name, values.last_name);
 })
 ```
+![Prompt](/docs/assets/img/api/dialog-api-prompt.png)
+*frappe.prompt*
 
-## frappe.confirm
+### frappe.confirm
 `frappe.confirm(message, if_yes, if_no)`
 
 Show a confirmation modal, executes `if_yes` if confirmation is given else
 executes `if_no`.
 
 ```js
-frappe.confirm('Are you sure you want to delete this document?',
+frappe.confirm('Are you sure you want to proceed?',
 	() => {
-		// delete document
+		// do something
 	}, () => {
 		// do nothing
 	})
-```
\ No newline at end of file
+```
+![Prompt](/docs/assets/img/api/dialog-api-confirm.png)
+*frappe.confirm*
+
+### frappe.show_alert
+`frappe.show_alert(message, seconds)`
+
+Alert Dialog is used for showing non-obstructive messages.
+
+Its parameters include the message to be shown and its display duration. The default is **3 seconds**.
+
+```js
+frappe.show_alert('Hi, you have a new message', 5);
+```
+
```

Comment:
frappe.show_alert also allows indicator. You can add some information regarding that.